docs/credentials/EcommercePackingListRegistrationCredential.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/traceability/v1",
+    {
+      "orderNumber": "https://schema.org/orderNumber"
```

Comment:
these terms should be defined in `https://w3id.org/traceability/v1`...

Reply:
@OR13 ... we talked about it out of band ... the term is defined in the schema but it is not top hierarchy such that it would show up in the context directly, but rather under "items" in an array ... apparently the JSON-LD checker cannot resolve properties in an array with defined entries. The Packing List is the only schema that as an array with a defined set of properties for each entry. 

Any suggestion as to how to resolve that issue? Open an issue

Will take care of the broken links, URLs, and naming right away. Apologies for the IRI slip. Forgot about it :-(

Reply:
@Therecanbeonlyone1969 
There is a trick to this that i used in [IntentToSell](https://github.com/w3c-ccg/traceability-vocab/blob/main/docs/schemas/IntentToSell.json)

effectively in the comment section where you are defining the `@ id` you have to have a unique value as it uses a struct to map over when building the json-ld from the schema, the easy way to do this is to add a query param that is meaningless, but unique within the object (similar to cache busting)

Reply:
removed